code/game/objects/structures/statues.dm

```diff
@@ -8,12 +13,14 @@
 	max_integrity = 100
 	var/oreAmount = 5
 	var/material_drop_type = /obj/item/stack/sheet/metal
+	var/impressiveness = 15
 	CanAtmosPass = ATMOS_PASS_DENSITY
 
 
 /obj/structure/statue/attackby(obj/item/W, mob/living/user, params)
 	add_fingerprint(user)
-	user.changeNext_move(CLICK_CD_MELEE)
+	if(!do_after(user, 20, target = src))
```

Comment:
What's this about

Reply:
oranges said to get rid of user.changeNext_move, see original PR

Reply:
Ah, i see. I feel to better represent the old functionality storing dustoff time would be better. And just show "no dust here yet" message only to the user if it's too soon after last one. (+ moodlet since i guess you don't need dust to appreciate art)

Reply:
I made these changes, I have no idea why it doesn't say outdated